source/content/guides/multisite/01-introduction.md

```diff
@@ -41,17 +41,19 @@ We do not support uses of WordPress Multisite that run functionally-different or
 ## Request a WordPress Multisite
 WordPress Multisite requires a special configuration that is only available to select customers. Access to WordPress Multisite requires a Gold, Platinum, or Diamond level [Workspace plan](https://pantheon.io/plans/pricing).
 
-By default, Pantheon employees must create new Multisites on your behalf. Reach out to your account manager to request site creation for a new WordPress Multisite.
+By default, Pantheon employees must create new Multisites on your behalf. This is because they use a different site [framework](/glossary/#framework) (which includes server settings like Nginx configuration) than the default `wordpress` framework typically available to customers. Reach out to your account manager to request site creation for a new WordPress Multisite.
 
 If you want to create new WordPress Multisites self-serve and on-demand in the Pantheon Dashboard, reach out to your account manager and request a [Custom Upstream](/guides/custom-upstream) for your workspace based on Pantheon's WordPress Multisite framework.
 
-Existing WordPress sites cannot be converted to a multisite, however they can be [migrated](/migrate-wordpress-multisite).
+Setting up an existing WordPress site  on Pantheon as a WordPress multisite without the correct framework can result in unexpected errors due to the improper Nginx configuration. If you have an existing WordPress site that you want to make into a multisite but the upstream does not use the `wordpress_network` framework (visible in the Pantheon Dashboard), and a WordPress Multisite upstream has been created for you, you can create a new site based on that upstream and push your codebase pushed to the new site repository (since WordPress Multisite is a core component of WordPress and uses the same code).
```

Comment:
```suggestion
Existing WordPress sites cannot be converted to WordPress Multisites (since they are different frameworks), however you can spin up a new WordPress Multisite and then [manually migrate](/migrate-manual) the code, database and files from your existing site. 
```